### DISCUSSION TITLE
Extend API search to cover operation name and endpoint name

### DISCUSSION THREAD

**Discussion by Mile76** [Azure/api-management-developer-portal#2601] [Feature request]
Hi, 

I would like to see the API text search that is currently available in the API portal to also cover endpoint names and operation names on the top level. Currently the search allows only to find matching API names. As some complex API contain many endpoints, a search down to the endpoint name and then showing the API with that endpoint would be nice.

Best regards
Michael

**Comment by ghost** [comment]
@Mile76, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@Mile76, thank you for requesting this feature.
